Common/src/Utils/ExpressionsBuilders.cs

```diff
@@ -23,8 +23,20 @@
 
 namespace ArmoniK.Core.Common.Utils;
 
+/// <summary>
+///   Provides utility methods for building LINQ expressions to filter data models based on field values.
+/// </summary>
 public static class ExpressionsBuilders
 {
+  /// <summary>
+  ///   Builds a filter expression for a field based on a list of values.
+  /// </summary>
+  /// <typeparam name="TData">The type of the data model.</typeparam>
+  /// <typeparam name="TField">The type of the field to filter.</typeparam>
+  /// <param name="expression">An expression representing the field to filter.</param>
+  /// <param name="values">The list of values to filter by.</param>
+  /// <param name="include">If true, includes the values in the filter; otherwise, excludes them.</param>
```

Comment:
Include is more about inverting the expression or not:
If true, the filter is inclusive, otherwise the filtered values are excluded from the result.

Same for the function below

Reply:
```suggestion
  /// <param name="include">If true, <paramref cref="filter" /> is whitelist, otherwise it is a blacklist.</param>
```